src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2661,6 +2661,8 @@ private ISymbolNode GetHelperFtnUncached(CorInfoHelpFunc ftnNum)
 
         private void* getHelperFtn(CorInfoHelpFunc ftnNum, ref void* ppIndirection)
         {
+            if (ppIndirection != null)
```

Comment:
This check is not necessary. It does not do what it appears to do since ppIndirection is `ref`.

I would assign it unconditionally at the end, like:
```
ppIndirection = null;
return (void*)ObjectToHandle(entryPoint);
```